src/panels/lovelace/common/handle-action.ts

```diff
@@ -76,14 +82,16 @@ export const handleAction = (
         forwardHaptic("light");
       }
       break;
-    case "call-service": {
+    case "call-service":
       if (!actionConfig.service) {
         forwardHaptic("failure");
         return;
       }
       const [domain, service] = actionConfig.service.split(".", 2);
       hass.callService(domain, service, actionConfig.service_data);
       forwardHaptic("light");
-    }
+      break;
+    case "custom":
+      fireEvent(node, "custom", actionConfig);
```

Comment:
These names are global and generic. If we want to do this, call it `ll-custom` to make clear that it's tied to Lovelace.